lib/heapwatch.js

```diff
@@ -30,7 +40,19 @@ class HeapWatch {
                 // Report memory stats to statsd. Use 'timing' (which isn't really
                 // timing-specific at all) instead of 'gauge' to get percentiles &
                 // min/max.
-                this.statsd.timing(`gc.${gcTypeName(stats.gctype)}`, stats.pause);
+                const type = gcTypeName(stats.gctype);
+                if (type !== 'unknown') {
+                    this.cumulativeGCTimes[gcTypeName(stats.gctype)] += stats.pause;
+                }
+                this.gcReportInterval = setInterval(() => {
```

Comment:
The reporting interval should be set only once, not every time the event fires.

Reply:
Done